src/test/Assertions.sol

```diff
@@ -1,6 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
```

Comment:
wait, wdym? 😅

wasn't the hole point to move them from `src/test/` in the `test/utils/`?

also to add `"test/utils"` in the published files:

<img width="228" alt="image" src="https://github.com/PaulRBerg/prb-math/assets/99738872/30d82dbd-647c-490a-80ff-4e180be983f2">

Reply:
@andreivladbrg see the comment in the file:

> This file is here for backward compatibility. It will be removed in V5

But anyway, this is de minimis. We could just remove the file and call it a day (for simplicity's sake).